data/verb-forms.json

```diff
@@ -254,6 +254,265 @@
+                    "ru": "говоришь"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "ssf",
+                "text": {
+                    "ar": "تَقُولِينَ",
+                    "en": "say",
+                    "ru": "говоришь"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "sdm",
+                "text": {
+                    "ar": "تَقُولَانِ",
+                    "en": "say",
+                    "ru": "говорите"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "sdf",
+                "text": {
+                    "ar": "تَقُولَانِ",
+                    "en": "say",
+                    "ru": "говорите"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "spm",
+                "text": {
+                    "ar": "تَقُولُونَ",
+                    "en": "say",
+                    "ru": "говорите"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "spf",
+                "text": {
+                    "ar": "تَقُلْنَ",
+                    "en": "say",
+                    "ru": "говорите"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "tsm",
+                "text": {
+                    "ar": "يَقُولُ",
+                    "en": "says",
+                    "ru": "говорит"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "tsf",
+                "text": {
+                    "ar": "تَقُولُ",
+                    "en": "says",
+                    "ru": "говорит"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "tdm",
+                "text": {
+                    "ar": "يَقُولَانِ",
+                    "en": "say",
+                    "ru": "говорят"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "tdf",
+                "text": {
+                    "ar": "تَقُولَانِ",
+                    "en": "say",
+                    "ru": "говорят"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "tpm",
+                "text": {
+                    "ar": "يَقُولُونَ",
+                    "en": "say",
+                    "ru": "говорят"
+                }
+            },
+            {
+                "tense": "present",
+                "pronoun": "tpf",
+                "text": {
+                    "ar": "يَقُلْنَ",
+                    "en": "say",
+                    "ru": "говорят"
+                }
+            }
+        ],
+        "type": "mu3tal"
```

Comment:
Let's call it `weak`?

Reply:
There are different types of weak. I will add later other types.

Reply:
Agreed.